Added support for autogenerated launch configs (UNR-700) (#672)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/toolbars.md

 ![Toolbar]({{assetRoot}}assets/screen-grabs/toolbar/toolbar-settings.png)

##### General

| Setting | Description |
| --- | --- |
| SpatialOS directory | If you're using a non-standard structure, you'll need to set this yourself. This is empty by default. If you leave it empty, it defaults to `<GameRoot>/../spatial`. |

##### Play In Editor Settings

| Setting | Description |
| --- | --- |

> | Delete Dynamically Spawned Entites | If enabled, deletes any dynamically spawned entities when taking down the running PIE session |

Reviewer: Is this a checkbox? 
Suggested text: 

> | Delete Dynamically Spawned Entities | If enabled, SpatialOS/the GDK/the Unreal Engine (delete as appropriate) deletes any dynamically spawned entities (in your game world?) when you end the PIE session. |

> | Delete dynamically spawned entities | If checked, the GDK deletes any dynamically spawned entities from your game world when you end the PIE session. |

Reviewer 2: "From your SpatialOS deployment" is the better wording, right?

Author: Yep, updating now

> | Delete dynamically spawned entites | If checked, the GDK (delete as appropriate) deletes any dynamically spawned entities (from your game world?) when you end the PIE session. |

Reviewer: ```suggestion
| Delete dynamically spawned entities | If checked, the GDK deletes any dynamically spawned entities from your game world when you end the PIE session. |
```

##### Launch

| Setting | Description |
| --- | --- |

> | Generate default launch config | If checked, a launch configuration will be generated by default when launching spatial through the toolbar. |

Reviewer: ```suggestion
| Generate default launch config | If checked, the GDK creates a launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) by default when you launch a local deployment through the toolbar. |
```

| Launch configuration | The [launch configuration file]({{urlRoot}}/content/glossary#launch-configuration-file) to use when running `spatial local launch` using the **Start** button. |
| Stop on exit |  If enabled, shuts down running deployments when you close the Unreal Editor. |

##### Snapshots

| Setting | Description |
| --- | --- |
| Snapshot path | Use this to specify the filepath of your [snapshot]({{urlRoot}}/content/glossary#snapshot). If you leave this empty, it defaults to `<GameRoot>/../spatial/snapshots`. |
| Snapshot file name |  Name of your snapshot file. |

> | Generate placeholder entities in snapshot | If checked, placeholder entities will be added to the snapshot on generation |

Reviewer: ```suggestion
| Generate placeholder entities in snapshot | If checked, the GDK adds placeholder entities to the snapshot when it is generated|
```

Reviewer: Do we need to update https://docs.improbable.io/unreal/alpha/content/generating-a-snapshot#placeholder-entities if this is now an option?

> | Generate placeholder entities in snapshot | If checked, the GDK adds placeholder entities to the snapshot when it is generated|

Reviewer: ```suggestion
| Generate placeholder entities in snapshot | If checked, the GDK adds [placeholder entities]({{urlRoot}}/content/generating-a-snapshot#placeholder-entities) to the snapshot when it is generated|
```
